src/engine/enginebuffer.cpp

```diff
@@ -105,6 +105,7 @@ EngineBuffer::EngineBuffer(QString group, ConfigObject<ConfigValue>* _config,
           m_bPlayAfterLoading(false),
           m_fRampValue(0.0),
           m_iRampState(ENGINE_RAMP_NONE),
+          m_dPrevSampleRate(44100.);
```

Comment:
This silently assumes that the Rubberband default is 44100. 
I think we should set it to 0.0 by default, to guarantee that  setScaleParameters is called on the first callback.